### ISSUE TITLE
Better logging of unsupported subprotocol

### ISSUE THREAD

**Issue by dbrgn** [saltyrtc/saltyrtc-server-python#51]
When specifying an unsupported subprotocol, the error msg is as follows:

    [2017-02-07 08:06:45.943390] NOTICE: saltyrtc.server: Unsupported sub-protocol 'None', dropping client

Would it be possible to actually print the string specified by the client? Or is that functionality hidden in the websockets implementation? (I tried to fix it for 10 minutes, but didn't find the correct place where the subprotocol string is parsed...)

**Comment by lgrahl** [comment]
AFAIK `None` stands for *no sub-protocol provided*. It should log the supplied sub-protocols. If it doesn't, that might be a bug.

**Comment by lgrahl** [comment]
@dbrgn Ping

**Comment by dbrgn** [comment]
Yes, I'm sure. If I remember correctly the client used v1.saltyrtc.org while the server used v0.saltyrtc.org.

**Comment by lgrahl** [comment]
Might be a limitation of the WebSocket implementation the server uses. Let me check that.
